adapter/repository/in-memory-simple/src/main/java/org/mobilitydata/gtfsvalidator/execution/InMemoryExecutionParameterRepository.java

```diff
@@ -0,0 +1,52 @@
+package org.mobilitydata.gtfsvalidator.execution;
```

Comment:
Seems like this should be in `package org.mobilitydata.gtfsvalidator.db` with the other repo implementation?